scripts/build-run-single-node.sh

```diff
@@ -32,5 +31,20 @@ ${BIN_PATH} collect-gentxs --home ${HOME_DIR}
 sed -i'.bak' 's#"tcp://127.0.0.1:26657"#"tcp://0.0.0.0:26657"#g' ${HOME_DIR}/config/config.toml
 sed -i'.bak' 's#"null"#"kv"#g' ${HOME_DIR}/config/config.toml
 
+# Register the validator EVM address
+{
+  # wait for block 1
+  sleep 20s
+
+  # private key: da6ed55cb2894ac2c9c10209c09de8e8b9d109b910338d5bf3d747a7e1fc9eb9
+  ${BIN_PATH} tx qgb register \
+    "$(${BIN_PATH} keys show validator --home "${HOME_DIR}" --bech val -a)" \
+    0x966e6f22781EF6a6A82BBB4DB3df8E225DfD9488 \
+    --from validator \
+    --home "${HOME_DIR}" \
+    --fees 30000utia -b block \
+    -y
```

Comment:
I tried to manually verify this PR b/c I was about to publish the same PR and the logs indicate that a valid tx never gets included. In other words, I repeatedly see:

```
11:02AM INF executed block height=9 module=state num_invalid_txs=0 num_valid_txs=0
```

and no block height has `num_valid_txs=1` so I don't think this works as expected.

Reply:
Really? I am using the same mechanism in orchestrator-relayer and seems to be working fine.
Let me retry it locally and come back to you, thanks

Reply:
```
5:19PM INF received proposal module=consensus proposal={"Type":32,"block_id":{"hash":"76851502993E4E13EE96104E1DE825722F71C0EF5901ED1B10E022A3B939430D","parts":{"hash":"F2D94AE80A1694D887043DF05DBC03B336619C92B7984B44CDD61F28F0477EF8","total":1}},"height":2,"pol_round":-1,"round":0,"signature":"8qJGtyACnDLoJrUitZvAaQr+zrIYy4S4G92NrSmO1H6YZTOWwU9BRuGuA+23/BgBdyLFId9my3jx3BHp1jS8BA==","timestamp":"2023-08-22T15:19:50.893142049Z"}
5:19PM INF received complete proposal block hash=76851502993E4E13EE96104E1DE825722F71C0EF5901ED1B10E022A3B939430D height=2 module=consensus
5:19PM INF finalizing commit of block hash={} height=2 module=consensus num_txs=1 root=83F414EA6AEACE70D10F06E8F33C59B64FB4F3CA2D773048B2544685B3989049
5:19PM INF minted coins from module account amount=28948873utia from=mint module=x/bank
5:19PM INF executed block height=2 module=state num_invalid_txs=0 num_valid_txs=1
5:19PM INF commit synced commit=436F6D6D697449447B5B3539203130332038332031343420353320313937203233203536203736203439203139392039203137203133322034322031343120313733203233352032343620353320313336203230203132342032313120313138203234203139392037302032392031373020313037203137375D3A327D
5:19PM INF committed state app_hash=3B67539035C517384C31C70911842A8DADEBF63588147CD37618C7461DAA6BB1 height=2 module=state num_txs=1
5:19PM INF indexed block exents height=2 module=txindex
code: 0
codespace: ""
data: 12300A2E2F63656C65737469612E7167622E76312E4D7367526567697374657245564D41646472657373526573706F6E7365
events:
- attributes:
  - index: true
    key: c3BlbmRlcg==
    value: Y2VsZXN0aWExa3g0ZmZwZDBrZnMzd3g3cHhubTd2OXpkdHVmMzdueWRhbDRjNHc=
  - index: true
    key: YW1vdW50
    value: MzAwMDB1dGlh
  type: coin_spent
- attributes:
  - index: true
    key: cmVjZWl2ZXI=
    value: Y2VsZXN0aWExN3hwZnZha20yYW1nOTYyeWxzNmY4NHoza2VsbDhjNWxwbmpzM3M=
  - index: true
    key: YW1vdW50
    value: MzAwMDB1dGlh
  type: coin_received
- attributes:
  - index: true
    key: cmVjaXBpZW50
    value: Y2VsZXN0aWExN3hwZnZha20yYW1nOTYyeWxzNmY4NHoza2VsbDhjNWxwbmpzM3M=
  - index: true
    key: c2VuZGVy
    value: Y2VsZXN0aWExa3g0ZmZwZDBrZnMzd3g3cHhubTd2OXpkdHVmMzdueWRhbDRjNHc=
  - index: true
    key: YW1vdW50
    value: MzAwMDB1dGlh
  type: transfer
- attributes:
  - index: true
    key: c2VuZGVy
    value: Y2VsZXN0aWExa3g0ZmZwZDBrZnMzd3g3cHhubTd2OXpkdHVmMzdueWRhbDRjNHc=
  type: message
- attributes:
  - index: true
    key: ZmVl
    value: MzAwMDB1dGlh
  - index: true
    key: ZmVlX3BheWVy
    value: Y2VsZXN0aWExa3g0ZmZwZDBrZnMzd3g3cHhubTd2OXpkdHVmMzdueWRhbDRjNHc=
  type: tx
- attributes:
  - index: true
    key: YWNjX3NlcQ==
    value: Y2VsZXN0aWExa3g0ZmZwZDBrZnMzd3g3cHhubTd2OXpkdHVmMzdueWRhbDRjNHcvMQ==
  type: tx
- attributes:
  - index: true
    key: c2lnbmF0dXJl
    value: T2VWWjJ4NHBFVlE0cldEMFM1U0VjNjlzSXIzb3hXd3E5ZlRKQjBxZU51VVlidXZTcHlqRUVnYi9EVHZnSytrempMZlFrMGNNYlNiZ3R1VkJWTTVLWlE9PQ==
  type: tx
- attributes:
  - index: true
    key: YWN0aW9u
    value: L2NlbGVzdGlhLnFnYi52MS5Nc2dSZWdpc3RlckVWTUFkZHJlc3M=
  type: message
gas_used: "66962"
gas_wanted: "210000"
height: "2"
info: ""
logs:
- events:
  - attributes:
    - key: action
      value: /celestia.qgb.v1.MsgRegisterEVMAddress
    type: message
  log: ""
  msg_index: 0
raw_log: '[{"msg_index":0,"events":[{"type":"message","attributes":[{"key":"action","value":"/celestia.qgb.v1.MsgRegisterEVMAddress"}]}]}]'
timestamp: ""
tx: null
txhash: E6EBA561A51AC03D11F665EE41B25640B562F00DE6EA2C4C21DDCB815E6165E2
5:19PM INF Ensure peers module=pex numDialing=0 numInPeers=0 numOutPeers=0 numToDial=10
5:19PM INF No addresses to dial. Falling back to seeds module=pex
5:20PM INF Timed out dur=10995.972819 height=3 module=consensus round=0 step=1
5:20PM INF received proposal module=consensus proposal={"Type":32,"block_id":{"hash":"0C55F77B4CD3A2962BC5F73B245CD2117C64BBB6B0B0F43C6192E6272327D5AE","parts":{"hash":"16F5EB160AF6BAD52CA663488C6818A2B59CA18E8DC0490D853CB1113D8B4AB7","total":1}},"height":3,"pol_round":-1,"round":0,"signature":"xUI5omv9kJvLHQIPR2xPLttc6m3C3R4V+bXBJySBiHI5vUSt4GIfAvoONjk979H+i+8QeN0LZ0q7KCv1IHhuCw==","timestamp":"2023-08-22T15:20:01.897046032Z"}
5:20PM INF received complete proposal block hash=0C55F77B4CD3A2962BC5F73B245CD2117C64BBB6B0B0F43C6192E6272327D5AE height=3 module=consensus
5:20PM INF finalizing commit of block hash={} height=3 module=consensus num_txs=0 root=3B67539035C517384C31C70911842A8DADEBF63588147CD37618C7461DAA6BB1
5:20PM INF minted coins from module account amount=27899081utia from=mint module=x/bank
5:20PM INF executed block height=3 module=state num_invalid_txs=0 num_valid_txs=0
5:20PM INF commit synced commit=436F6D6D697449447B5B323138203137392031353120313231203137392032303520353720313932203133352032333120313937203839203136362035362031303820313633203739203233362037352031353220313620363820323136203136302031363120313833203138332031323720393520323033203633203137385D3A337D
5:20PM INF committed state app_hash=DAB39779B3CD39C087E7C559A6386CA34FEC4B981044D8A0A1B7B77F5FCB3FB2 height=3 module=state num_txs=0
5:20PM INF indexed block exents height=3 module=txindex
```

Seems to be included:
```
5:19PM INF executed block height=2 module=state num_invalid_txs=0 num_valid_txs=1
5:19PM INF commit synced commit=436F6D6D697449447B5B3539203130332038332031343420353320313937203233203536203736203439203139392039203137203133322034322031343120313733203233352032343620353320313336203230203132342032313120313138203234203139392037302032392031373020313037203137375D3A327D
5:19PM INF committed state app_hash=3B67539035C517384C31C70911842A8DADEBF63588147CD37618C7461DAA6BB1 height=2 module=state num_txs=1
```

Reply:
I'm not seeing the logs associated with the tx submission b/c that command is running in the background.

Reply:
```
code: 0
codespace: ""
data: 12300A2E2F63656C65737469612E7167622E76312E4D7367526567697374657245564D41646472657373526573706F6E7365
events:
- attributes:
  - index: true
    key: c3BlbmRlcg==
    value: Y2VsZXN0aWExa3g0ZmZwZDBrZnMzd3g3cHhubTd2OXpkdHVmMzdueWRhbDRjNHc=
  - index: true
    key: YW1vdW50
    value: MzAwMDB1dGlh
  type: coin_spent
- attributes:
  - index: true
    key: cmVjZWl2ZXI=
    value: Y2VsZXN0aWExN3hwZnZha20yYW1nOTYyeWxzNmY4NHoza2VsbDhjNWxwbmpzM3M=
  - index: true
    key: YW1vdW50
    value: MzAwMDB1dGlh
  type: coin_received
- attributes:
  - index: true
    key: cmVjaXBpZW50
    value: Y2VsZXN0aWExN3hwZnZha20yYW1nOTYyeWxzNmY4NHoza2VsbDhjNWxwbmpzM3M=
  - index: true
    key: c2VuZGVy
    value: Y2VsZXN0aWExa3g0ZmZwZDBrZnMzd3g3cHhubTd2OXpkdHVmMzdueWRhbDRjNHc=
  - index: true
    key: YW1vdW50
    value: MzAwMDB1dGlh
  type: transfer
- attributes:
  - index: true
    key: c2VuZGVy
    value: Y2VsZXN0aWExa3g0ZmZwZDBrZnMzd3g3cHhubTd2OXpkdHVmMzdueWRhbDRjNHc=
  type: message
- attributes:
  - index: true
    key: ZmVl
    value: MzAwMDB1dGlh
  - index: true
    key: ZmVlX3BheWVy
    value: Y2VsZXN0aWExa3g0ZmZwZDBrZnMzd3g3cHhubTd2OXpkdHVmMzdueWRhbDRjNHc=
  type: tx
- attributes:
  - index: true
    key: YWNjX3NlcQ==
    value: Y2VsZXN0aWExa3g0ZmZwZDBrZnMzd3g3cHhubTd2OXpkdHVmMzdueWRhbDRjNHcvMQ==
  type: tx
- attributes:
  - index: true
    key: c2lnbmF0dXJl
    value: T2VWWjJ4NHBFVlE0cldEMFM1U0VjNjlzSXIzb3hXd3E5ZlRKQjBxZU51VVlidXZTcHlqRUVnYi9EVHZnSytrempMZlFrMGNNYlNiZ3R1VkJWTTVLWlE9PQ==
  type: tx
- attributes:
  - index: true
    key: YWN0aW9u
    value: L2NlbGVzdGlhLnFnYi52MS5Nc2dSZWdpc3RlckVWTUFkZHJlc3M=
  type: message
gas_used: "66962"
gas_wanted: "210000"
height: "2"
info: ""
logs:
- events:
  - attributes:
    - key: action
      value: /celestia.qgb.v1.MsgRegisterEVMAddress
    type: message
  log: ""
  msg_index: 0
raw_log: '[{"msg_index":0,"events":[{"type":"message","attributes":[{"key":"action","value":"/celestia.qgb.v1.MsgRegisterEVMAddress"}]}]}]'
timestamp: ""
tx: null
txhash: E6EBA561A51AC03D11F665EE41B25640B562F00DE6EA2C4C21DDCB815E6165E2
```

These are the logs of the submission. Even if it is run in the background, it still outputs to stdout.

Or, could it be that this script does not work as expected in MacOS?

Reply:
I am on MacOS. Just retried and still not seeing any logs for the transaction submission or a block with 1 valid tx. Full logs [here](https://pastebin.com/MF7fQKq9).

Reply:
I see on line 19 this:

```
usage: sleep seconds
```

Could it be that sleep is not recognized and making the register block fail?

Coule you try removing the second suffix and run it again?

```
Sleep 20
```

?

Reply:
Good catch! I didn't see that log line. 

```
sleep 20
```

works for me!